### ISSUE TITLE
Pages can be hard to follow without side bar

### ISSUE THREAD

**Issue by addertheblack** [docker/docker.github.io#5011]
File: [engine/userguide/index.md](https://docs.docker.com/engine/userguide/), CC @mstanleyjones 

Because your user document pages don't always show with a sidebar there's absolutely no context for pages like this:

https://docs.docker.com/engine/userguide/

Steps to reproduce:

1) Go to link above
2) increase font size until sidebar along the left hand side disapears.
3) Image you're someone who has just search for documentation and lands on that page without a side bar - what the heck do you click on next? There's no way to navigate the documentation! I clicked on thumbs down because there's no way to do anything.

Using Firefox 56 on the lastest MacOS.. High sierra? Somethign like that.

**Comment by shin-** [comment]
FWIW You can show the sidebar using the button in the top-right corner.

![image](https://user-images.githubusercontent.com/1086876/31690419-8110224a-b347-11e7-9c32-762db029edbb.png)

**Comment by mstanleyjones** [comment]
I had to zoom 4 times to get this behavior on my 13" Macbook and I suspect most users are viewing on a 21"+ monitor. Considering that you basically get into a mobile view and the hamburger menu exists (as @shin- pointed out), I think this is working as well as expected.

**Comment by addertheblack** [comment]
I have a 22" 1080p monitor and get into mobile mode when I have web browser windows side by side or otherwise sharing screen space. I also have a 21" 1080p screen in portrait view, which is setup to view documents like this. I have a 13" Macbook screen and get into mobile mode because I read docs with a large font. This to say that there are multiple routes that seem to plunk the web site in mobile mode and these are just the ones I run into on my current setup. 

The mobile mode navigation isn't intuitive.  I would have never guessed the hamburger menu on the right side of the screen would have been the table of contents. Menus on the right side of the screen as typically related to you account profile or are otherwise site-wide navigation related. Right now in this git hub window the top right is populated by notification related items, repository management and my profile. gmail is setup similarly. In my experience navigation element go from document specific to general as you go out from the content or go from left to right. This impression is reinforced by having a set of tabs that switch to different types of documents and the hamburger menu line up with the same background color.. presumably grouping all site navigation together.

In any case, I didn't find the document navigation and until you mentioned it assumed it was a faulty web page rending. If I'm doing that, it's likely others are as well.

**Comment by johndmulhausen** [comment]
Now that you know about the hamburger menu, @addertheblack, is the proposed action on our end simply that the hamburger menu be moved to the left?

**Comment by sarahpark** [comment]
>is the proposed action on our end simply that the hamburger menu be moved to the left?

+1, I recommend moving the icon to the left + making the side nav overlay in mobile sizes (cc/ @jsouth for UX advice for Docs, though)

Examples:

<kbd>

![youtube](https://user-images.githubusercontent.com/1562758/32026807-ab32315e-b99a-11e7-9758-32e2dead1806.gif)

</kbd>

<kbd>

![he3b5axhep](https://user-images.githubusercontent.com/1562758/32026818-b4442e00-b99a-11e7-8f04-5429ce2a7db2.gif)

</kbd>

**Comment by addertheblack** [comment]
@johndmulhausen it might be that simple. 

I'd change the icon from a hamburger to an icon that implied a table of contents and not a system menu:

[This google search returned a few of what I mean](https://www.google.com/search?safe=off&tbm=isch&source=hp&biw=1277&bih=894&ei=-Bb9WfWYDYKBU6bGmiA&q=table+of+contents+icon&oq=table+of+contents+icon&gs_l=img.3..0l2.1933.5409.0.5568.24.19.0.0.0.0.324.2640.0j9j4j1.14.0....0...1.1.64.img..10.14.2635.0...0.LGwt3OR-QVg).

[Here's a direct link to what I mean](https://d30y9cdsu7xlg0.cloudfront.net/png/1010520-200.png)

Ebook.com's ebook use this icon. Google's "Play Reader" ebook reader on Android uses the left handed variant.


@sarahpark you  can add the Android version of Amazon Kindle ebook reader to your list. I noticed it on the way home.

cheers guys..

**Comment by londoncalling** [comment]
@johndmulhausen this may be an easy issue to knock out, starting with moving hamburger menu to left and/or labeling it TOC.

**Comment by L-Hudson** [comment]
Closing this ticket due to its age, and the impending refactor. If you think this is in error, feel free to reopen. Thanks!
